packages/shared/src/components/modals/RanksModal.tsx

```diff
@@ -1,73 +1,19 @@
-import React, {
-  CSSProperties,
-  ReactElement,
-  ReactNode,
-  useContext,
-} from 'react';
+import React, { CSSProperties, ReactElement, useContext } from 'react';
```

Comment:
Let's split this file into multiple files. It has so many things going on here. We can create a directory for the `RanksModal` and have everything there

Reply:
On it 🥳